src/backend/common/CMakeLists.txt

```diff
@@ -76,6 +76,10 @@ target_link_libraries(afcommon_interface
     ${CMAKE_DL_LIBS}
 )
 
+if(AF_BUILD_FORGE)
+  add_dependencies(afcommon_interface forge)
```

Comment:
This doesn't add link dependency in the linker, I have verified this on Linux and Windows. This approach works on both the OSes unlike the earlier one which weirdly doesn't work on Windows.